packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -1685,17 +1685,103 @@ describe('postgresql partitioned tables introspection', () => {
+                                                                                                                                                        // 
+                                                                                                                                                        // The following models were ignored as they do not have a valid unique identifier or id. This is currently not supported by the Prisma Client.
+                                                                                                                                                        // - "definitely_not_measurement"
+                                                                                                                                                        // - "measurement"
+                                                                                                                                                        // 
+                                                                                                                                                        // These tables are partition tables, which are not yet fully supported.
+                                                                                                                                                        // - Model "definitely_not_measurement"
+                                                                                                                                                        // - Model "measurement"
+                                                                                                                                                        // 
+                                                                                                                  `)
+      expect(ctx.mocked['process.stdout.write'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+      expect(ctx.mocked['process.stderr.write'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+    })
+  })
+})
+
+describe('postgresql partitioned tables introspection', () => {
+  const connectionString = process.env.TEST_POSTGRES_URI_MIGRATE!.replace('tests-migrate', 'tests-migrate-db-pull')
+
+  const computeSetupParams = (warningCode: number, variant?: number): SetupParams => {
+    const setupParams: SetupParams = {
+      connectionString,
+      // Note: dirname points to a location with a setup.sql file
+      // which will be executed to prepare the database with the correct tables, views etc.
+      dirname: path.join(
+        __dirname,
+        '..',
+        '__tests__',
+        'fixtures',
+        'introspection',
+        'postgresql',
+        `row-level-security-warning-${warningCode}${variant ? `-${variant}` : ''}`,
+      ),
+    }
+    return setupParams
+  }
+
+  const setupPostgressForWarning = (warningCode: number, variant?: number) => {
```

Comment:
```suggestion
  const setupPostgresForWarning = (warningCode: number, variant?: number) => {
```
or is the `ss` intentional?

Reply:
Ope, good catch, it was unintentional